app/example_app/books.py

```diff
@@ -0,0 +1,112 @@
+from flask import render_template, request, redirect, flash
+from .. import db
+from ..models import  Book, Announcement, Event, Graphic, Borrow
+from . import example_app
+
+@example_app.route("/Employee")
+def employee():
+    return render_template("/books/employee.html")
+
+@example_app.route('/Books', methods=['POST','GET'])
+def books():
+    if request.method == 'POST':
+        nazwa = request.form['nazwa']
+        strony = request.form['strony']
+        if nazwa != "" and not nazwa.isspace():
+            if strony.isdigit():
+                nowa_ksiazka = Book(nazwa=nazwa, strony=strony)
+            else:
+                nowa_ksiazka = Book(nazwa=nazwa)
+            try:
+                db.session.add(nowa_ksiazka)
+                db.session.commit()
+            except:
+                return "Problem z dodaniem ksiazki"
+        else:
+            flash('Blad dodania ksiazki(nazwa nie moze byc pusta)')
+        return redirect("/Books")
+    else:
+        ksiazki = Book.query.order_by(Book.id).all()
+        return render_template("/books/books.html",ksiazki=ksiazki)
+
+@example_app.route("/Books/delete/<int:id>")
+def deleteBook(id):
+    book_to_delete = Book.query.get_or_404(id)
+
+    try:
+        db.session.delete(book_to_delete)
+        db.session.commit()
+        return redirect("/Books")
+    except:
+        return "Blad przy usuwaniu ksiazki"
+
+@example_app.route("/Books/update/<int:id>",methods=['POST','GET'])
+def updateBook(id):
```

Comment:
Konwencja nazewnictwa metod/funkcji w Pythonie.
```suggestion
def update_book(id):
```